### ISSUE TITLE
Rubocop executable broken after changes in #1850

### ISSUE THREAD

**Issue by devinrm** [w0rp/ale#1920]
## Information

**VIM version**

0.3.1

Operating System: 
MacOS 10.13.6

### :ALEInfo

that's outputting nothing at the moment and crashing nvim

## What went wrong

After updating this is what happens when opening a ruby file:
![screen shot 2018-09-14 at 10 19 08](https://user-images.githubusercontent.com/13222919/45556153-94c52f80-b808-11e8-8a52-9981732bc1f5.png)

Rubocop still works from the command line. Tried uninstalling and reinstalling ale and uninstall and reinstalling rubocop

ALE config:
```
let g:ale_linters = {
      \ 'ruby': ['ruby', 'rubocop', 'rails_best_practices', 'reek', 'brakeman']
      \ }
```

## Reproducing the bug

1. Update to latest ALE
2. Open ruby file

**Comment by Draiken** [comment]
Same thing happening for me on Arch Linux

**Comment by devinrm** [comment]
Closing because this was fixed #1921 Thanks!
